let x=10;
try{
    console.log("hii")
}catch(error){
    console.log("error occured")
}finally{
    console.log("completed")
}

throw "tobig coding"